src/Algebra/Module/Properties.agda

```diff
@@ -0,0 +1,97 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions of algebraic structures defined over some other
+-- structure, like modules and vector spaces
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Module.Properties where
+
+open import Algebra
+open import Algebra.Module
+open import Level
+
+private
+  variable
+    c ℓ : Level
+
+self-leftSemimodule : (R : Semiring c ℓ) → LeftSemimodule R c ℓ
```

Comment:
Why not name these `semiring⇒leftSemimodule`? The `self` seems confusing, and it implies that semirings are the only stuctures which construct trivial semimodules.